packages/@glimmer/syntax/test/loc-node-test.ts

```diff
@@ -291,6 +291,18 @@ data-barf="herpy"
   }
 });
 
+test("element dynamic attribute", function() {
+  let ast = parse(`<img src={{blah}}>`);
+
+  let [div] = ast.body;
```

Comment:
Can you rename this to local variable to `img` (since it is referencing the `img` not a `div`)?